.gitignore

```diff
@@ -0,0 +1,3 @@
+# Vim related files
+tags
+*.swp
```

Comment:
It feels like these ignores are not necessarily related to this repo, have you considered putting them in a global gitignore file on your machine instead?